### DISCUSSION TITLE
Does geopandas.read_file support reading the MDB format?

### DISCUSSION THREAD

**Discussion by awa5114** [geopandas/geopandas#3476] [Q&A]
With `ogr2ogr` I am able to convert an ESRI Personal GeoDataBase file to a GPKG as follows:

```
ogr2ogr -f GPKG db.gpkg --config PGEO_DRIVER_TEMPLATE="DRIVER={Microsoft Access Driver (*.mdb, *.accdb)};DBQ=C:\Users\admin\Desktop\TEMP\projects\314\db.mdb;" db.mdb
```

I ftried to open this file using various `gpd.read_file` calls, but they all throw errors ranging from `Failed to initialize DSN to ODBC connection` to `Unsupported driver PGeo`. I've tried setting the `pyogrio` engine explicitly and passing the `PGEO_DRIVER_TEMPLATE` config option using a `config_option={"PGEO_DRIVER_TEMPLATE": "DRIVER={Microsoft Access Driver..."}` all to no avail.

Does geopandas support MDB as claimed by @m-richards in [this question](https://github.com/geopandas/geopandas/discussions/3465#discussioncomment-11251885)?

**Comment by m-richards** [comment]
@amine-aboufirass I can't say I'm familar with every single driver, it was more a suggestion that if something works with ogr2ogr it likely works directly with geopandas.read_file as gdal is the common underlying tool - and if you were able to confirm it works with your file that might be useful information for a future user trying to work with mdb files. 

Having a quick look though, I don't have any substantial mdb files on hand, but for example reading this mdb file from the gdal tests works just fine:
```python
import geopandas as gpd
import requests

open(r"./geometry_types.mdb", "wb").write(requests.get("https://github.com/OSGeo/gdal/raw/refs/heads/master/autotest/ogr/data/pgeo/geometry_types.mdb").content)

gdf = gpd.read_file(r"./geometry_types.mdb", layer='polygon_z')
print(gdf.dtypes)
print(gdf.crs)

```

Note this might also depend on how you've obtained geopandas and dependencies (conda vs pip), here I'm using pyogrio as the io engine, installed by conda-forge on windows.
